.devcontainer/Dockerfile

```diff
@@ -1,20 +1,13 @@
-FROM ghcr.io/n64-tools/gcc-toolchain-mips64:latest
+FROM ubuntu:latest
```

Comment:
I would always suggest a preference for debian slim here, unless there is a particular reason (as the image is a lot smaller).

Reply:
Fixed in branch to use debian slim.